src/components/PathwayCourse/index.js

```diff
@@ -466,20 +531,40 @@ function PathwayCourse() {
               </Grid>
             ))}
           </Grid>
-          {/* <Grid  sx={{mb:15}}align="center">
+          <Grid onClick={completedAll ? handleModal : handleSnackbar} sx={{mb:15}}align="center">
             <Grid sx={{mb:3}} >
               <img src={require("./asset/separator.svg")} alt="icon" />
             </Grid>
-            <Grid>
-              <img
-                src={require("./asset/certificate.svg")}
-                alt="certificate icon"
-              />
+            <Grid sx={{cursor: "pointer"}}>
+              {
+                completedAll ? 
+                <CertificateIconColored
+                  className={classes.certificateIcon}
+                /> : 
+                <CertificateIcon
+                  className={classes.certificateIcon}
+                />
+              }
               <Typography sx={{mt:2}} variant="body1" mb={2}>
-                {pathwayCourseData?.pathway} Certificate (Locked)
+                {pathwayCourseData?.pathway} Certificate
               </Typography>
             </Grid>
-          </Grid> */}
+            <Snackbar
+              open={openSnackbar}
+              autoHideDuration={6000}
+              message={`Please complete all the courses to unlock ${pathwayCourseData?.pathway}  certificate`}
```

Comment:
Should use message constants via the `Message` component here to allow support for other languages. (https://github.com/navgurukul/bhanwari-devi/blob/main/src/components/common/Message/index.js)

Reply:
I guess translation happens with the help of keys in the `msg` folder. How will it translate a sentence? Haven't really used `Message` before.

Reply:
Yes, and you can set the constant to be a complete sentence or phrase. If you need to include a placeholder for a JavaScript expression such as you do here with `pathwayCourseData?.pathway`. just use %1 for the first such placeholder, %2 for the second, etc. in the string literal. Then you can pass the value you want to use for it in the `args` prop. So, for example, here you'd use something like `en.CERTIFICATE_COMPLETION_WARNING = `Please complete all the courses to unlock %1 certificate` in [en.js](https://github.com/navgurukul/bhanwari-devi/blob/main/src/msg/en.js) and the Hindi translation in `hi.js`. Then could use the following here:
```suggestion
              message={<Message constantKey="CERTIFICATE_COMPLETION_WARNING" args={[pathwayCourseData?.pathway]} />}
```

Reply:
Thanks for the crisp explanation!